tests/Language/LanguageServiceTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @copyright Copyright (c) Wizacha
+ * @license Proprietary
+ */
+declare(strict_types = 1);
+
+namespace Wizaplace\SDK\Tests\Language;
+
+use Wizaplace\SDK\Cms\CmsService;
+use Wizaplace\SDK\Language\LanguageService;
+use Wizaplace\SDK\Tests\ApiTestCase;
+
+final class LanguageServiceTest extends ApiTestCase
+{
+    /**
+     * @var LanguageService
+     */
+    private $languageService;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->languageService = new LanguageService($this->buildAdminApiClient());
+    }
+
+    public function testGetAllLanguages()
+    {
+        $languages = $this->languageService->getAllLanguages();
+
+        $this->assertSame(2, count($languages));
+
+        $french = $languages[0];
+
+        $this->assertSame('Français', $french->getName());
+        $this->assertSame('fr', $french->getLangCode());
+        $this->assertSame('FR', $french->getCountryCode());
+        $this->assertSame('active', $french->getStatus());
```

Comment:
tous les appels à PHPUnit sont statiques. PHP autorise l'utilisation de `$this`, mais ça n'a pas de logique en statique